administrator-documentation/moderne-platform/how-to-guides/lst-cleanup.md

```diff
@@ -0,0 +1,36 @@
+# Cleaning up old LSTs
+
+As your team develops new code, LSTs will be created and added to your artifact repository. Over time, however, many of these LSTs will no longer be useful. This is especially common when working in feature branches that get merged in or deleted – all LSTs for those branches serve no point as any changes made on them can't be merged in at that point.
```

Comment:
So there are two different but similar scenarios where LST deletion is desirable.

You describe the defunct branch scenario here, when a branch is no longer relevant to development and you _could_ run recipes against it but there would be no point. To address this the old LST needs to be deleted from the customer's artifact repository, _then_ the graphql mutation needs to be run.

There's also the "old LSTs for still in-use branches" cleanup scenario. In that scenario the branch is still viable and no graphql mutation needs to be run to use the latest version of the LST. In that case any older LSTs for the same repository+branch are purely a waste of storage space.